### ISSUE TITLE
[Feature] Delete Messages / Conversation / Contact - consistency between clients.

### ISSUE THREAD

**Issue by keybreak** [oxen-io/session-android#1350]
### Is there an existing request for feature?

- [X] I have searched the existing issues

### What feature would you like?

Instead of just **Delete** option for chat / group, we should be consistent with desktop version and do:
- Delete Messages
- Delete Conversation
- Delete Contact

It's very important and straight-forward, each function is needed on any Session client.

**Comment by KeeJef** [comment]
Yep, we will be making this consistent across platforms in the closed group update
